lib/node_modules/@stdlib/blas/ext/base/sdsnansum/docs/types/index.d.ts

```diff
@@ -1,7 +1,7 @@
 /*
 * @license Apache-2.0
 *
-* Copyright (c) 2020 The Stdlib Authors.
+* Copyright (c) 2024 The Stdlib Authors.
```

Comment:
```suggestion
* Copyright (c) 2020 The Stdlib Authors.
```

This file was not modified. No need to change the copyright year.